utilities_common/bgp_util.py

```diff
@@ -215,8 +215,10 @@ def get_bgp_summary_from_all_bgp_instances(af, namespace, display):
         except ValueError:
             ctx.fail("bgp summary from bgp container not in json format")
 
+        # exit cli command without printing the error message
         if key not in cmd_output_json:
-            ctx.fail("bgp summary from bgp container in invalid format")
+            click.echo("% No IP{} neighbor is configured".format(af))
```

Comment:
Please remove "% " (% and the space after it) as that is not in the original error message or the service that needs to parse it will not match...
Thanks!

Reply:
Fixed